charts/ocis/docs/values.adoc.yaml

```diff
@@ -1229,6 +1229,9 @@ services:
       sha: ""
       # -- Image pull policy
       pullPolicy:
+    # -- Maximum number of concurrent go-routines. Higher values can potentially get work done faster but will also cause 
+    # more load on the system. Values of 0 or below will be ignored and the default value will be used.
+    maxConcurrency: "1"
```

Comment:
frontend has this setting already in 

https://github.com/owncloud/ocis-charts/blob/fcbfec935673da46bc8e617366e2c97fd6b6e629/charts/ocis/values.yaml#L1187-L1188